projects/angular/src/data/datagrid/datagrid-virtual-scroll.directive.ts

```diff
@@ -262,6 +284,22 @@ export class ClrDatagridVirtualScrollDirective<T> implements AfterViewInit, DoCh
     });
   }
 
+  clearItems() {
```

Comment:
Can we rename it to something more obvious and also make it private? We can also leave a comment explaining that this is generating a "placeholder" for the total items inside the virtual scroll
```suggestion
  private populateVirtualData() {
```

Reply:
Making it `private` make sense, but renaming it not so much.

Suggested name `populateVirtualData` implies putting something inside, but actually the method is reseting all items to `empty` vlaues, therefore the name `clearItems`.

`resetVirtualData` or `clearVirtualData` are good name candidates.

Reply:
`populatePlaceholderData`

Reply:
Commit ccb73ba